src/resources/elements/primeDesignSystem/pform-input/pform-input.ts

```diff
@@ -0,0 +1,53 @@
+import {BindingEngine, child, customElement} from "aurelia-framework";
+import "./pform-input.scss";
+import {bindable, observable} from "aurelia-typed-observable-plugin";
+import {ValidationState} from "../types";
+import {Disposable} from "aurelia-binding";
+
+@customElement("pform-input")
+export class PFormInput {
+  @bindable.string @observable validationState?: ValidationState;
+  @bindable.string label = "";
+  @bindable.string labelDescription = "";
+  @bindable validationMessage = "";
+  @bindable.boolean showCounter = false;
+  @bindable.number maxValue = 0;
+  @bindable.string helperMessage = "";
+  @child("*") input;
```

Comment:
Wondering if this part needs a comment, or is it self-explanatory.

Looking at the html code, might make sense to add a comment, because we use named-slots as well.